README.md

```diff
@@ -3,8 +3,10 @@
 ## Supported Platforms
```

Comment:
Hmm, do we want to soften this to "Available Platforms"?  We don't support libvirt (ever) or the experimental platforms (yet) in the "Red Hat Support" sense.